mobile/src/main/java/ch/epfl/sdp/mobile/ui/Api.kt

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.sdp.mobile.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import ch.epfl.sdp.mobile.data.api.AuthenticationApi
+
+/** A global composition local which provides access to an instance of [AuthenticationApi]. */
+val LocalAuthenticationApi = compositionLocalOf<AuthenticationApi> { error("Missing API.") }
+
+/**
+ * Provides the given APIs through different [androidx.compose.runtime.CompositionLocal] values
+ * available throughout the hierarchy.
+ *
+ * @param authentication the [AuthenticationApi] that will be provided.
+ */
+@Composable
+fun ProvideApis(
```

Comment:
Suggestion : maybe change the filename to a more specific one that `Api.kt`?

Reply:
Sure thing, I'll rename it to `ApiCompositionLocals.kt`, to better reflect that the code only relates to providing some composition locals for the different `***Api.kt` we will have.

Reply:
Done in 2dbf458370d3705d49308c8a47038ef50fec9c52.